setup/security/authentication/oauth/index.md

```diff
@@ -206,6 +206,17 @@ is never revealed outside of the server using it.
 
 {% include mermaid %}
 
+## Restricting access based on User Info
+
+User access can be restricted further based on the user info from an OAuth ID token. This
+requirement is set via the `--user-info-requirements` parameter. This enables us to restrict user
+login to specific domains or having a specific attribute. Use equal signs between key and value,
+and additional key/value pairs need to repeat the flag.
+```
+# Example:
+hal config security authn oauth2 edit --user-info-requirements hd=google.com
```

Comment:
Maybe use an example that includes multiple key-value pairs, to make it even clearer to repeat the flag?

Reply:
IIRC someone added the ability to use regex in those reqs also.

Reply:
Instead of `google.com` in the example, I'd say something like `my-org.net` or something that indicates the user's domain.

Reply:
Done, added multiple values, added notes on regex.